### DISCUSSION TITLE
Saving volume and playback quality settings

### DISCUSSION THREAD

**Discussion by AgentSmith0** [vidstack/player#988] [Q&A] [answered]
Is it possible for vidstack to store things like volume and playback quality settings globally in persistent storage, like Cookies or LocalStorage?

**Comment by mihar-22** [comment, marked as answer]
It's one of those things that I think is an application-level concern but we can consider trying to make it easier. For the time being you'll need to add it yourself.

**Reply by AgentSmith0** [reply]
@mihar-22 Hello, how can I change the volume or quality (maybe using the MediaRemoteControl class) when I use vidstack through CDN?

**Reply by mihar-22** [reply]
You can dispatch the media request event:

```ts

el.dispatchEvent(new CustomEvent('media-volume-change-request', {
  detail: 0.5, // 50%
  bubbles: true
}))
```

You can find the event for each remote method in the [`MediaRemoteControl` API docs](https://www.vidstack.io/docs/player/api/classes/media-remote-control?styling=default-theme#changevolume).

**Reply by AgentSmith0** [reply]
Thanks!
